### ISSUE TITLE
ERROR: LoadError: LoadError: ArgumentError: Mux not found in path

### ISSUE THREAD

**Issue by ackalker** [JunoLab/Blink.jl#18]
Trying to install and use Blink package in a freshly installed Julia gives me the error as mentioned in the title.
These are the steps I took:
```julia
julia> Pkg.add("Blink")
INFO: Initializing package repository /home/miki/.julia/v0.4
[...snip build and download log...]
julia> using Blink
INFO: Precompiling module Reexport...
INFO: Precompiling module Lazy...
INFO: Precompiling module JSON...
ERROR: LoadError: LoadError: ArgumentError: Mux not found in path
while loading /home/miki/.julia/v0.4/Blink/src/content/content.jl, in expression starting on line 1
while loading /home/miki/.julia/v0.4/Blink/src/Blink.jl, in expression starting on line 6
```
To resolve, I did:
```julia
julia> Pkg.add("Mux")
```
I guess that Blink is missing a dependency on Mux.

**Comment by rohitvarkey** [comment]
Resolved by https://github.com/JunoLab/Blink.jl/pull/15. The version containing the updated REQUIRE hasn't been tagged yet, which is why the error still exists on a `Pkg.add("Blink")`.

**Comment by ackalker** [comment]
Thanks for your very quick reply :)

**Comment by rohitvarkey** [comment]
@spencerlyon2 Can you tag a new version with the REQUIRE update?

**Comment by spencerlyon2** [comment]
Yes, will do. 

@Varanas I'll tag a patch now and if we need another hotfix for #20 we can tag a patch after that also

**Comment by spencerlyon2** [comment]
Closed by #15
